libs/datasets/taglib.py

```diff
@@ -293,6 +294,23 @@ def content(self) -> str:
         return json.dumps(d, separators=(",", ":"))
 
 
+@dataclass(frozen=True)
+class KnownIssueNoDate(TagInTimeseries):
+    disclaimer: str
```

Comment:
Should we be internally consistent with the codebase and call this "public_note" (we can consider renaming it in the json file)?